src/style.css

```diff
@@ -1 +1,25 @@
 /* styles go here */
+.fieldset {
```

Comment:
in this task u can use only 2 class (or more, but JFYI)

Reply:
Ok, now understand, but do I need to change it in this task or I can leave it